api/services/Slack.js

```diff
@@ -0,0 +1,32 @@
+var format = require('util').format
+var Promise = require('bluebird')
+var request = require('request')
+var post = Promise.promisify(request.post)
+
+module.exports = {
+  textForNewPost: function (post, community) {
+    var relatedUser, creator = post.relations.creator
+
+    if (post.isWelcome()) {
+      relatedUser = post.relations.relatedUsers.first()
+      return format('<%s|%s> joined <%s|%s>',
```

Comment:
by the way, now that we've switched to using Node 5, we can use template strings instead of `format` :)